packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx

```diff
@@ -519,7 +535,7 @@ function ActiveVisitsTable() {
           </Button>
         </Tile>
       </div>
-      {showOverlay && <PatientSearch view={view} closePanel={() => setShowOverlay(false)} />}
```

Comment:
You should also update this to launch the search extension as it breaks
```
<p className={styles.content}>{t('noPatientsToDisplay', 'No patients to display')}</p>
 <Button
     kind="ghost"
     size="sm"
     renderIcon={(props) => <Add size={16} {...props} />}
      onClick={() => setShowOverlay(true)}>
      {t('addPatientToQueue', 'Add patient to queue')}
</Button>
```

Reply:
@CynthiaKamau, @hadijahkyampeire Kindly I have fixed the proposed changes.